CRM/Core/SelectValues.php

```diff
@@ -1177,4 +1177,17 @@ public static function optionValueFields() {
     ];
   }
 
+  /**
+   * Limit-to options for schedule reminders.
+   *
+   * @return array
+   */
+  public static function getLimitToValues() {
+    return [
+      1 => ts('Neither'),
+      2 => ts('Limit or Add Recipients'),
+      3 => ts('Also include'),
```

Comment:
For consistency of these first letter case options, minor edit to change it to "Also Include"